script.js

```diff
@@ -1,4 +1,224 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+//Scissors, paper, stone
+var userWinCount = 0;
+var computerWinCount = 0;
+var drawsCount = 0;
+var userWinningPercentage = 0;
+var computerWinningPercentage = 0;
+var totalGameCount = 0;
+
+var calculatePercentage = function (winCount, totalCount) {
+  return ((winCount / totalCount) * 100).toFixed(2);
+};
+
+var gameMode = function (userInput) {
+  var mode = "invalid";
+
+  if (userInput == "scissors" || userInput == "paper" || userInput == "stone") {
+    mode = "normal";
+  } else if (
+    userInput == "reversed scissors" ||
+    userInput == "reversed paper" ||
+    userInput == "reversed stone"
+  ) {
+    mode = "reverse";
+  }
+
+  return mode;
+};
+
+var testingConditions = function (userChoice, computerChoice) {
+  if (
```

Comment:
i think this function can be named slightly better? what conditions are you testing for if you want to name it this way?